Jenkinsfile

```diff
@@ -12,22 +12,18 @@ node('CLIENT') {
         checkout scm
         sh '''git submodule update --init'''
 
-    stage 'Qt5'
-        sh '''rm -rf build
+    stage 'Linux'
+        def linux = docker.image('dominikschmidt/owncloud-client-ci-image:latest')
```

Comment:
ping @tboerger to get this image into https://github.com/owncloud-ci - the windows build image as well

Reply:
Obviously ... as stated in the PR text ;-)

Reply:
This keeps being the long term goal. We had a lengthly discussion with @tboerger , @jnweiger and @ogoffart. 
Basically: If we want to move this to owncloud-ci, we need to base it on Ubuntu LTS. That's problematic because the current Ubuntu LTS has a too old Qt Version and the next one will only be released in April. We want to continue on this, but for now we want to stick to my image - so we can actually get a bit of work done now.